Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_20.md

```diff
@@ -0,0 +1,26 @@
+
+#### Integrations
+
+##### Cortex Attack Surface Management
+- Updated the Docker image to: *demisto/python3:3.10.12.63474*.
+#### Playbooks
+
+##### New: Cortex ASM - Remediation Confirmation Scan
+
+- Created a new playbook for Remediation Confirmation Scan (RCS) creation and polling. (Available from Cortex XSOAR 6.8.0).
+
+##### Cortex ASM - ASM Alert
+
+- Updated the playbook with new formatting and removed step for verifying Xpanse scan results with NMAP.
+
+##### Cortex ASM - Detect Service
+
+- Updated the playbook by replacing NMAP, SNMP, and S3 logic with Cortex ASM - Remediation Confirmation Scan playbook.
+
+##### Cortex ASM - AWS Enrichment
+
+- Added the "AWS - Unclaimed S3 Bucket Validation" playbook for support with AWS S3 remediation.
+
+##### Cortex ASM - SNMP Check
+
+- Deprecated the playbook.
```

Comment:
```suggestion
Deprecated.
```

Reply:
@ShirleyDenkberg that does not use one of our templates. It must be `Deprecated. ` with a space after specifically I think.

https://xsoar.pan.dev/docs/documentation/release-notes#general

Reply:
Actually neither `Deprecated. ` or `Deprecated.` works.

```
 - Notes for the line: "Deprecated."
   - Line is not using one of our templates, consider changing it to fit our standard.
```